index.js

```diff
@@ -901,6 +902,42 @@ class Hyperdrive extends Nanoresource {
     }
   }
 
+  mirror () {
+    const self = this
+    if (this._mirrorRanges) return unmirror
+    this._mirrorRanges = new Map()
+
+    this.on('content-feed', oncore)
+    this.on('metadata-feed', oncore)
+    // Getting all the mounts will trigger the event listeners above.
+    this.getAllMounts({ content: true }, (err, mounts) => {
```

Comment:
Does getAllMounts reutn the metadata/content of the target drive here as well? Otherwise we need to add them if the events above have fired earlier

Reply:
Yeah you mean the top-level drive? Yeah those will be included

Reply:
👍